fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraVersioningIT.java

```diff
@@ -620,10 +621,25 @@ public void testGetTimeMapResponseForBinaryDescription() throws Exception {
         verifyTimemapResponse(descriptionUri, descriptionId);
     }
 
+    /**
+     * Verify an application/link-format TimeMap response.
+     *
+     * @param uri The full URI of the Original Resource.
+     * @param id The path of the Original Resource.
+     * @throws Exception
```

Comment:
JavaDoc wants a description on the `Exception` as well. Such as:
`@throws Exception on HTTP request error`